spec/revision-2/schema.json

```diff
@@ -0,0 +1,109 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Cosmic Reach Modpack Schema",
+	"description": "Schema for a cosmic reach modpack file",
+	"type": "object",
+	"required": ["spec_revision", "name", "loader", "version", "files"],
+	"properties": {
+		"spec_revision": {
+			"type": "number",
+			"description": "The revision of the spec your modpack is using"
+		},
+		"name": {
+			"type": "string",
+			"description": "The name of your modpack"
+		},
+		"version": {
+			"type": "string",
+			"description": "The version identifier of the modpack"
+		},
+		"description": {
+			"type": "string",
+			"description": "The description of the modpack (Optional)"
+		},
+		"icon": {
+			"type": "string",
+			"description": "The URL of the modpack's icon (Optional)"
```

Comment:
May not be necessary, but it may be a good idea to say that it could be a path from the root of the zip/folder (ik saying that it is a "URL" let's them know that, but directly stating it cannot hurt).